docs/explanations/about-ddd.md

```diff
@@ -0,0 +1,26 @@
+# About Domain-Driven Design (DDD)
```

Comment:
Wikipedia writes "Domain-driven design" instead of "About Domain-Driven Design".